ang/civicase-base.ang.php

```diff
@@ -12,8 +12,26 @@
 use CRM_Civicase_Helper_OptionValues as OptionValuesHelper;
 use CRM_Civicase_Helper_GlobRecursive as GlobRecursive;
 use CRM_Civicase_Helper_NewCaseWebform as NewCaseWebform;
+use CRM_Civicase_Helper_CaseCategory as CaseCategoryHelper;
+
+$caseCategoryId = CRM_Utils_Request::retrieve('case_type_category', 'Int');
+if ($caseCategoryId > 0) {
+  $caseCategoryName = civicrm_api3('OptionValue', 'getsingle', [
+    'option_group_id' => 'case_type_categories',
+    'value' => $caseCategoryId,
+    'return' => ['name'],
+  ])['name'];
+}
+else {
+  $caseCategoryName = CaseCategoryHelper::CASE_TYPE_CATEGORY_NAME;
```

Comment:
Are we still supporting string values?

Reply:
Not from the URL, but the value is still required on other parts of the code. In other words, it will not work with the param `case_type_category=Cases`, but the name is required for other logic, then calculated.